src/assets/js/throttle.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
This may make more sense as one of many exports in a utils instead of its own file.

Reply:
I am not a fan of the concept of multiple utility functions living in one file when they do not require interoperability with each other. Keeping functions in discretely named files helps make them more searchable and more portable from one project to another.

Furthermore, this method keeps files concise functionally and reduces the chance of messy Git diffs where changes in one function appear as changes in an unrelated one because they have similar structure or syntax.

Reply:
Move it into a utils folder then. I'd rather not have all those files live at the same spot. Ideally, we could then alias that too so we don't have as long of an import path (and to make it consistently imported).

Reply:
I have added an `@scripts` alias to the webpack config and updated the import path. This PR created the directory `assets/js` so this is the only file in this there. This is the same structurally as the css directory. I have not changed the throttle.js file location